go/tools/go-upgrade/go-upgrade.go

```diff
@@ -393,6 +393,9 @@ func replaceGoVersionInCodebase(old, new *version.Version, workflowUpdate bool)
 		"./test/templates",
 		"./build.env",
 		"./docker/bootstrap/Dockerfile.common",
+		"./docker/lite/Dockerfile",
+		"./docker/lite/Dockerfile.percona80",
+		"./docker/vttestserver/Dockerfile.mysql80",
```

Comment:
We now have to bump the golang version in more places since we base more of our images on the official go docker image.